openreview/conference/matching.py

```diff
@@ -430,6 +430,20 @@ def _build_config_invitation(self, scores_specification):
                         'description': 'Invitation to store paper user assignments',
                         'order': 12
                     },
+                    'custom_user_demand_invitation': {
+                        'description': 'Invitation to store custom max number of users required by papers',
+                        'default': self.match_group.id + '/-/Custom_Max_Users',
+                        'order': 13,
+                        'value-regex': self.match_group.id + '/-/.*',
+                        'required': False
+                    },
+                    'custom_max_papers_invitation': {
```

Comment:
Doesn't this invitation replace the current 'custom_load_invitation'?

Reply:
Yes, both serve the same purpose. Which do you prefer to keep? I would keep the new one i.e. `custom_max_papers_invitation` just because the name and description are clearer.

Reply:
Yes, I agree